### PR TITLE
Fix: Update variant calling tool - data type table in docs

### LINKED ISSUE #803: Can be tumor-only somatic variant calling performed using Strelka?
### Description of the bug

As you mention in your Nextflow pipeline: @https://nf-co.re/sarek/3.0.2/parameters it would be possible to obtain the results of tumor-only somatic variant calling using strelka tool (among other methods). 

I've run the pipeline using strelka with tumor-only samples and I've ended up having results for germline variants (variants.vcf.gz, genome.S${N}.vcf.gz) instead of somatic ones (somatic.snvs.vcf.gz, somatic.indels.vcf.gz) (@https://github.com/Illumina/strelka/blob/v2.9.x/docs/userGuide/README.md ).

I read that strelka can perform somatic variant calling when you have tumor-normal samples pairs, however it is not possible to do so when you have tumor-only samples.  @https://academic.oup.com/bioinformatics/article/28/14/1811/218573. Did I miss any extra parameter?

Sorry in advance if i misunderstood something.

### Command used and terminal output

_No response_

### Relevant files

_No response_

### System information

_No response_

### PR DESCRIPTION
**Pull request by famosab**
## PR checklist

Closes #803

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

I updated Normal -> Germline, Tumor -> Tumow-Only, Somatic -> Somatic (Tumor-Normal) to clarify the docs. I changed Strelka to not being able to process tumor-only samples and MSI to being able to process tumor-only.

### CODE REVIEW

**Comment by bounlu** [comment]
In the current pipeline, `MSIsensorpro` doesn't work with tumor-only samples, unless you merge my [PR](https://github.com/nf-core/sarek/pull/1633).

**Review by maxulysse — APPROVED** [review]

File: docs/usage.md [hunk 1]
```diff
@@ -575,20 +575,20 @@ Sarek can be started at different points in the analysis by setting the paramete
...
+| [Manta](https://github.com/Illumina/manta)                                                              |  x  |  x  |   x    |     x     |     x      |           x            |
+| [TIDDIT](https://github.com/SciLifeLab/TIDDIT)                                                          |  x  |  x  |   x    |     x     |     x      |           x            |
+| [ASCAT](https://github.com/VanLoo-lab/ascat)                                                            |  x  |  x  |   -    |     -     |     -      |           x            |
+| [CNVKit](https://cnvkit.readthedocs.io/en/stable/)                                                      |  x  |  x  |   -    |     x     |     x      |           x            |
+| [Control-FREEC](https://github.com/BoevaLab/FREEC)                                                      |  x  |  x  |   x    |     -     |     x      |           x            |
+| [MSIsensorPro](https://github.com/xjtu-omics/msisensor-pro)                                             |  x  |  x  |   x    |     -     |     x      |           x            |
```
Comment (maxulysse) [inline docs/usage.md:591]:
```suggestion
| [MSIsensorPro](https://github.com/xjtu-omics/msisensor-pro)                                             |  x  |  x  |   x    |     -     |     -      |           x            |
```

Comment (famosab) [reply docs/usage.md:591]: Should we wait for that then?

File: CHANGELOG.md [hunk 2]
```diff
@@ -98,6 +98,7 @@ Ruopsokjåkhå is another peak of the Pårte massif.
 
 - [1623](https://github.com/nf-core/sarek/pull/1623) - Update docs to clarify vep cache folder organisation
 - [1628](https://github.com/nf-core/sarek/pull/1628) - Fix dbsnp channel mapping in germline variant calling subworkflow
+- [1635](https://github.com/nf-core/sarek/pull/1635) - Fix docs to reflect variant calling tool - data type correctly
```
Comment (FriederikeHanssen) [inline CHANGELOG.md:101]:
```suggestion
```

**Review by FriederikeHanssen — APPROVED** [review]

**Review by maxulysse — APPROVED** [review]

**Pull request merged**